src/model.js

```diff
@@ -25,6 +25,21 @@ const _group = 'Context';
 
 const _propFlags = GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT;
 
+/*
+ * List of BaseApps can be found by searching for 'BaseApp' in the flathub project
+ * https://github.com/flathub?utf8=%E2%9C%93&q=baseApp&type=&language=
+ */
+const _blacklist = [
+    'io.elementary.BaseApp ',
+    'io.qt.qtwebkit.BaseApp',
+    'io.liri.BaseApp',
+    'io.qt.qtwebengine.BaseApp',
+    'org.electronjs.Electron.BaseApp',
+    'org.electronjs.Electron2.BaseApp',
+    'org.godotengine.godot.BaseApp',
+    'org.mozilla.Firefox.BaseApp'
```

Comment:
I don't think maintaining a hard coded list of app-ids would scale well. But, we could add a `isBaseApp()` method though, which could translate to ` return appId.endsWith('.BaseApp')`.

A bit of a hack, but I think is a decent trade-off considering `.BaseApp` is a well established convention at this point.